xtask/src/qemu.rs

```diff
@@ -359,6 +359,8 @@ pub fn run_qemu(arch: UefiArch, opt: &QemuOpt) -> Result<()> {
 
     if arch == UefiArch::IA32 || arch == UefiArch::X86_64 {
         cmd.args(["-debugcon", "file:./integration-test-debugcon.log"]);
+        cmd.args(["-chardev", "file,id=fw,path=./ovmf-firmware-debugcon.log"]);
+        cmd.args(["-device", "isa-debugcon,chardev=fw,iobase=0x402"]);
```

Comment:
I'm not sure if this works?! And we missed to add the file to `.gitignore´. I'Ve opened #1640